src/pages/prep-page/PrepPage.jsx

```diff
@@ -1,9 +1,34 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
 import "./PrepPage.css";
 
-function PrepPage() {
+function PrepPage({ quizName, link }) {
+  const background = {
+    backgroundImage: 'url("/src/assets/images/cool-background.png")',
+    backgroundRepeat: "no-repeat",
+    backgroundSize: "cover",
+  };
+  const quiz = quizName ? quizName : "this Quiz";
+
   return (
-    <div>PrepPage</div>
-  )
+    <div
+      className="d-flex justify-content-center align-items-center h-100"
+      style={background}
+    >
+      <div
+        className="bg-light rounded h-25 p-4 d-flex flex-column align-items-center"
+        style={{ width: "50%", minWidth: "20rem" }}
+      >
+        <h1 className="mb-4">Ready for {quiz}?</h1>
+        <Link className="w-50 h-50" to={quiz}>
```

Comment:
So for the quiz in the "to" property should be in quotation marks and it should have a forward slash in front or the navigation won't work

Reply:
Hardcoded "to" prop to "/questions" to link to questions page